plugins/scaffolder/src/components/MultistepJsonForm/schema.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { JsonObject } from '@backstage/config';
+import { FormProps } from '@rjsf/core';
+
+function isObject(value: unknown): value is JsonObject {
```

Comment:
hehe should this also go in `@backstage/config`? ;)

Reply:
One of the utils that I don't care if it exists in every file xP